pkgs/by-name/ze/zenmap/package.nix

```
@@ -0,0 +1,76 @@
+{
+  gobject-introspection,
+  gtk3,
+  lib,
+  nmap,
+  python3Packages,
+  wrapGAppsHook3,
+  xterm,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "zenmap";
+  version = nmap.version;
+  pyproject = true;
+
+  src = nmap.src;
+
+  sourceRoot = "${nmap.name}/zenmap";
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  buildInputs = [
+    nmap
+    gtk3
+    xterm
+  ];
+
+  nativeBuildInputs = [
+    wrapGAppsHook3
+    gobject-introspection
+  ];
+
+  dependencies = with python3Packages; [
+    pygobject3
+  ];
+
```

```suggestion
  nativeCheckInputs = [
    nmap
  ];
```
Required by the test suite